libraries/default.rb

```diff
@@ -1042,7 +1007,7 @@ def install_publish_1x_service_config(current_deploy_root, enabled, engage_url)
         group 'opencast'
         variables({
           publish_1x_engage_url: engage_url,
-          publish_1x_enabled: enabled 
+          publish_1x_enabled: enabled
```

Comment:
At some point we should remove all the 1x stuff.  We have been on 5x a long time.